### DISCUSSION TITLE
Love Salvo! Please take a quick look at my Salvo REST API...

### DISCUSSION THREAD

**Discussion by geospatialweb** [salvo-rs/salvo#720] [Show and tell]
I have a development sandbox website where I try new things in a learning environment. Recently, I have been learning Rust and I decided to convert a Go REST API to a Rust REST API using Salvo. I love Salvo's simplicity, intuitiveness and elegance - particularly for a Rust novice such as myself. The docs are great and I was able to integrate a number of the examples including TLS, CORS and JWT Auth.

https://gitlab.com/geospatialweb/rust-mvt-postgis
I welcome your feedback...

John Campbell

**Comment by josejachuf** [comment]
Hi @geospatialweb 

Just in a few days I am to be released (open source), a Tile Vector server, using Salvo.

**Reply by geospatialweb** [reply]
Hi @josejachuf

That sounds really interesting! I would love to give it go when it is released...

**Reply by josejachuf** [reply]
Hi @geospatialweb, 
We are using it with a layer of more than 2.000.000 polygons and the behavior is great.

**Reply by geospatialweb** [reply]
Hi @josejachuf 

Wow! That's really impressive... Reads PostGIS geometry directly I assume... Do you know of any remote work opportunities where I could apply this solution?

**Reply by geospatialweb** [reply]
Hi @josejachuf

Any word on the release of your Tile Vector server? Like to give it test drive...

**Reply by josejachuf** [reply]
Hi @geospatialweb 

I sent you an email, please let me know if you received it.
